toolchain/check/decl_name_stack.cpp

```diff
@@ -88,19 +88,19 @@ auto DeclNameStack::Suspend() -> SuspendedName {
       << "Missing call to FinishName before Suspend";
   SuspendedName result = {.name_context = decl_name_stack_.pop_back_val(),
                           .scopes = {}};
-  auto enclosing_index = result.name_context.initial_scope_index;
+  auto parent_index = result.name_context.initial_scope_index;
```

Comment:
Maybe `initial_index`? This isn't necessarily a parent of any particular scope.

Reply:
Done with `scope_index` because I feel better removing the first word than the middle one, and this is just a shorthand local variable anyways.